### ISSUE TITLE
An unreasonable place in the experiment

### ISSUE THREAD

**Issue by kaqiz** [hengruizhang98/CCA-SSG#7]
In the experimental code
`if val_acc >= best_val_acc:  best_val_acc = val_acc if test_acc > eval_acc: eval_acc = test_acc` 
Isn’t this unreasonable? Normally, save the best  validation set  to calculate the results of the test set. If test_acc > eval_acc is added, isn’t it equivalent to face-to-face answer parameter adjustment?

**Comment by hengruizhang98** [comment]
You are right. See this issue: https://github.com/hengruizhang98/CCA-SSG/issues/2. I will fix the codes shortly.

**Comment by kaqiz** [comment]
Thank you for your reply. I would like to know if the data of the comparison model in your paper was also generated using the same method. If I use the changed code as an indicator, will the data of the comparison model also need to be changed? Thank you.

**Comment by kaqiz** [comment]
> You are right. See this issue: #2. I will fix the codes shortly.

Thank you for your reply. I would like to know if the data of the comparison model in your paper was also generated using the same method. If I use the changed code as an indicator, will the data of the comparison model also need to be changed? Thank you.

**Comment by hengruizhang98** [comment]
Sorry for the late reply. The baselines' results are correct, you may directly use the results in our paper for convenience.

**Comment by kaqiz** [comment]
> Sorry for the late reply. The baselines' results are correct, you may directly use the results in our paper for convenience.

Thank you for your reply. I have one last question. There is a certain degree of fluctuation in the experimental data in your paper. Is this fluctuation caused by the 20 experiments themselves, or is it caused by changes in model parameters. Also, you wrote in your paper that We adopt the public splits for Cora, Citeseer, Pubmed, and a 1:1:9 training/validation/testing splits for the other 4 datasets. But It is 1:1:8 in the code,Right?

**Comment by hengruizhang98** [comment]
For Cora, Citeseer, and Pubmed, the fluctuation is caused by the model initialization since the split is fixed. For other datasets, it is caused by both the random split and model initialization. The correct split is 1:1:8. (1:1:9 is a typo).

**Comment by kaqiz** [comment]
> For Cora, Citeseer, and Pubmed, the fluctuation is caused by the model initialization since the split is fixed. For other datasets, it is caused by both the random split and model initialization. The correct split is 1:1:8. (1:1:9 is a typo).

Thanks a lot! I read the latest code,your code is ` elif val_acc == best_val_acc and test_acc > eval_acc
                eval_acc = test_acc`,I think it is just `eval_acc = test_acc`,because test can not see the val. 
best wishes!

**Comment by hengruizhang98** [comment]
Thanks for your suggestions.
